ci/kokoro/docker/build-in-docker-bazel.sh

```diff
@@ -197,12 +132,12 @@ if [[ "${RUN_INTEGRATION_TESTS}" == "yes" || \
     echo
     echo "================================================================"
     log_normal "Delete service account used in the storage HMAC tests."
```

Comment:
s/Delete/Activate/?

Reply:
Both actually, created additional log lines to make this clearer.